From f712d0139e3a0b1146566c989a3d523da9cdcd42 Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Tue, 7 Jun 2016 20:01:55 +0100
Subject: [PATCH 1/3] fix application icon does not show on GNOME desktop

- phoenix.pro renames fritzing_icon.png to fritzing.png
- Icon value should be either an absolute path or icon name
---
 fritzing.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fritzing.desktop b/fritzing.desktop
index 854319eb..1791c928 100644
--- a/fritzing.desktop
+++ b/fritzing.desktop
@@ -4,7 +4,7 @@ Name=Fritzing
 GenericName=Fritzing
 Comment=Electronic Design Automation software
 Exec=Fritzing
-Icon=icons/fritzing_icon.png
+Icon=fritzing
 Terminal=false
 Type=Application
 Categories=Development;IDE;Electronics;EDA;

From a64635049f62234e03344da5be6f8d3bf09e2072 Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Tue, 7 Jun 2016 20:01:55 +0100
Subject: [PATCH 2/3] fix desktop Categories entry

Single entry prefixing non-standard categories
---
 fritzing.desktop | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/fritzing.desktop b/fritzing.desktop
index 1791c928..5b14537d 100644
--- a/fritzing.desktop
+++ b/fritzing.desktop
@@ -7,8 +7,7 @@ Exec=Fritzing
 Icon=fritzing
 Terminal=false
 Type=Application
-Categories=Development;IDE;Electronics;EDA;
+Categories=Development;IDE;Electronics;X-EDA;X-PCB;
 X-SuSE-translate=false
 StartupNotify=true
-Categories=PCB;
 MimeType=application/x-fritzing-fz;application/x-fritzing-fzz;application/x-fritzing-fzp;application/x-fritzing-fzpz;application/x-fritzing-fzb;application/x-fritzing-fzbz;application/x-fritzing-fzm;

From 789149760451defdf55a27647fcf1c20105cb2f8 Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Tue, 7 Jun 2016 20:01:55 +0100
Subject: [PATCH 3/3] remove desktop Version entry

Version is the version of the Desktop Entry Specification not Fritzing
---
 fritzing.desktop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fritzing.desktop b/fritzing.desktop
index 5b14537d..6b88de20 100644
--- a/fritzing.desktop
+++ b/fritzing.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Version=0.9.3b
 Name=Fritzing
 GenericName=Fritzing
 Comment=Electronic Design Automation software
